log4go_test.go

```diff
@@ -128,9 +105,48 @@ func TestFileLogWriter(t *testing.T) {
 	defer os.Remove(testLogFile)
 
 	w.LogWrite(newLogRecord(CRITICAL, "source", "message"))
+	w.LogWrite(newLogRecord(CRITICAL, "source", "message"))
+	w.Close()
+	runtime.Gosched()
+
+	if contents, err := ioutil.ReadFile(testLogFile); err != nil {
+		t.Errorf("read(%q): %s", testLogFile, err)
+	} else if len(contents) != 100 {
+		t.Errorf("malformed filelog: %q (%d bytes)", string(contents), len(contents))
+	}
+
+	w = NewFileLogWriter(testLogFile, false)
+	w.SetRotate(true)
+	w.SetRotateLines(5)
+
+	// Already has one line from write above
```

Comment:
Two lines?

Reply:
Ah yes, the source of my disdain for random comments: they can become outdated even after only a few hours